#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_check_smtp_help.dpatch by Jan Wagner <waja@cyconet.org>
##
## DP: Add -F to help output (bugs.debian.org/578333)
## DP: Upstream bug is: https://sourceforge.net/tracker/?func=detail&aid=2989193&group_id=29880&atid=397597

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' nagios-plugins-1.4.14~/plugins/check_smtp.c nagios-plugins-1.4.14/plugins/check_smtp.c
--- nagios-plugins-1.4.14~/plugins/check_smtp.c	2009-06-06 09:04:49.000000000 +0200
+++ nagios-plugins-1.4.14/plugins/check_smtp.c	2010-04-19 10:54:02.000000000 +0200
@@ -782,6 +782,8 @@
   printf ("    %s\n", _("Expected response to command (may be used repeatedly)"));
   printf (" %s\n", "-f, --from=STRING");
   printf ("    %s\n", _("FROM-address to include in MAIL command, required by Exchange 2000")),
+  printf (" %s\n", "-F, --fqdn=STRING");
+  printf ("    %s\n", _("FQDN used for HELO"));
 #ifdef HAVE_SSL
   printf (" %s\n", "-D, --certificate=INTEGER");
   printf ("    %s\n", _("Minimum number of days a certificate has to be valid."));
@@ -823,8 +825,8 @@
 print_usage (void)
 {
   printf (_("Usage:"));
-	printf ("%s -H host [-p port] [-e expect] [-C command] [-f from addr]", progname);
+  printf ("%s -H host [-p port] [-e expect] [-C command] [-f from addr]\n", progname);
   printf ("[-A authtype -U authuser -P authpass] [-w warn] [-c crit] [-t timeout]\n");
-  printf ("[-S] [-D days] [-v] [-4|-6]\n");
+  printf ("[-F fqdn] [-S] [-D days] [-v] [-4|-6]\n");
 }
 
